news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,126 @@
+
+If however you find suspicious behaviors, you can
+ - [Report this as a bug on GitHub](https://github.com/PrestaShop/PrestaShop/issues) (read [how to report issues](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/))
+ - Submit a bug fix by creating a [pull request](https://github.com/PrestaShop/PrestaShop/compare) (read the [contribution guidelines](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/))
+
+# Release Candidate
+
+When Beta period ends, we consider that all known regressions for this release are now registered into the Kanban. So the aim is clear: fix it all and we can ship.
+
+We fix the remaining regressions (or merge the fixes submitted by the community) and maintainers deliver a Release Candidate Build using the `1.7.7.x` branch codebase. This will be the Release Candidate 1 also known as RC1.
+
+This Build is provided to everyone, and we then start the **timer**. We wait for one week. During this week we continue testing and exploring the Build, trying to find anything that would not have been detected earlier and the community should do the same.
+
+By the end of this week, if no new regression has been reported, the RC1 becomes the final Release and **PrestaShop 1.7.7.0 is out !**
+
+... However, I dont think it ever happened than RC1 was flawless.
+
+Most of the time, some more regressions will be reported. When a new regression is reported, we fix it and we reset the timer. So there is a RC2, and if more regressions are reported, there is a RC3... until no new regressions are reported.
+
+Finally the latest built Release Candidate becomes the stable PrestaShop 1.7.7.0 software release.
+
+# Summary
+
+- The cycle begins when we start working on a minor version scope. When we complete the scope of this version, we trigger the Feature Freeze.
+
+Expected duration: 4 months
+
+- Following Feature Freeze an Alpha build is delivered and tested by QA team. QA team will then report all known defects and we will be able to plan then deliver a Beta build.
+
+Expected duration: 1 month
+
+- The Beta Build is open to everybody and we host a one-month long period for both maintainers and community to test the build.
+
+Expected duration: 1 month
+
+- At the end of the Beta period, we fix all the bugs that have been found and maintainers deliver a Release Candidate 1 build.
+After a week, if no bugs have been reported in RC1 build, we deliver the stable release. However if bugs a reported, we fix them and deliver a RC2, and so on until we build a version with no reported issues. This last Release Candidate becomes the stable release.
+
+Expected duration: from 1 week (if RC1 is flawless) to 1 month, possibly 2 months
+
+### Calendar
+
+If you sum all expected phases duration, you reach 6 months. This is why we expect to deliver 2 minor releases by year. 2 phases of development, 2 phases of feature freeze and 2 beta periods.
```

Comment:
```suggestion
The global duration for all the process is about 6 months. This is why we expect to deliver 2 minor releases by year.
```
Removed the last part because I found it confusing